fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -367,4 +368,17 @@ public void setIntegrityUnchecked(String schemaName, String tableName) {
         // not expecting this to be called for this adapter
         throw new UnsupportedOperationException("Set integrity unchecked not supported for this adapter.");
     }
+    
+    @Override
+    public void alterTableColumnIdentityCache(String schemaName, String tableName, String columnName, int cache) {
+        // Not supported by PostgreSQL
+        
+        final String qname = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
+        DataDefinitionUtil.assertValidName(columnName);
+        
+        // modify the CACHE property of the identity column
+        final String ddl = "ALTER TABLE " + qname + " ALTER COLUMN " + columnName + " SET CACHE " + cache;
+
+        warnOnce(MessageKey.ALTER_TABLE_SEQ_CACHE, "Not supported in PostgreSQL: " + ddl);
```

Comment:
is there a way to support it in PostgreSQL?
do we think we won't have the problem there?

Reply:
Right, I was thinking about this. We need to see if it's also a problem, but because it means having to fish through the catalog to find the sequence, it's going to be a fair bit of work. Now, I'd still like to migrate to the new schema design which eliminates the row_id identity completely, thus rendering this issue moot.